extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -354,8 +309,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     /** @private {boolean} */
     this.isFluid_ = false;
 
-    /** @private {?string} */
-    this.fluidImpressionUrl_ = null;
+    /** {?string} */
```

Comment:
Revert

Reply:
Need to access this actually, I don't think I can revert